### ISSUE TITLE
Add ‘confusable’ characters validation to Payment modal and to user display name creation

### ISSUE THREAD

**Issue by alicjakujawa** [JoinColony/colonyDapp#3186]
## Specification
* Use created in separate ticket validation method in Payment modal 
* Display warning if user name dont pass validation
* Use this validation method also on user display name creation

### Design (optional, might not be applicable)
https://www.figma.com/file/Phby7kEYihhvGgQn69A314/Verified-Recipients?node-id=79%3A11161

**Comment by alicjakujawa** [comment]
Blocked by https://github.com/JoinColony/colonyDapp/issues/3185
